trunner/long_test.py

```diff
@@ -0,0 +1,140 @@
+#
```

Comment:
Please do not name it as `long_test.py`. Maybe create directory named `harnesses` and there we can have `base.py` for base classes, `unity.py` for Unity, `busybox.py` for Busybox and `mbedtls.py` for Mbedtls tests.

Reply:
I will prefer to treat `busybox`, `mbedtls` and other tests of our ports as `long_tests` and place their harnesses in one file, but I think the idea of creating `harnesses` directory could be a good solution.

Reply:
The goal here was to make possible adding new test suite by changing possibly one file. 

EDIT: I am not sure about splitting it into files, like `unity.py`, `busybox.py`, because the only thing in those files will be one class with one harness method.